docs/UserGuide.md

```diff
@@ -529,6 +542,30 @@ is shown in the screenshot above. In the actual application, you can scroll down
 </div>
 
 ### 3. Commands - Manpower Management <a name="commands-manpower-management"></a>
+
+The Manpower Management features allows you to keep a record of your employees' data (i.e. employee's names,
+employee's phone numbers, tags that indicate what days the employee is available to work, etc) inside
+Employee Directory. You are able to view data related to manpower in the Employee Directory inside the _Main
+ View_.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about available drinks:**<br>
+
+* All employees who are currently working in the shop are active (unarchived) employees.
+
+* Employees who are no longer working in the shop can be deleted from the Employee Directory (refer to 
+ `c-delete` command for details), or moved to the archived Employee Directory (refer to `c-archive` command for
+  details).
+
+* All active (unarchived) employees' information is stored in the active Employee Directory (refer to `c-active-list
+` to view the active Employee Directory).
+
+* All archived employees' information is stored in the archived Employee Directory (refer to `c-archive-list`
+ to view the archived Employee Directory).
```

Comment:
Not sure if this would be considered repetitive information if they will be explained again later?

Reply:
I will delete the explanation  from the later part, and refer to this one instead. Thanks